config/locales/translations/zendesk_apps_support.yml

```diff
@@ -219,6 +219,30 @@ parts:
       key: "txt.apps.admin.warning.app_build.bitmap_in_svg"
       title: "App builder job: warning that svg contains an embedded bitmap image and cannot be used"
       value: "%{svg} contains an embedded bitmap and cannot be used as an app icon. It has been replaced with a default placeholder icon."
+  - translation:
+      key: "txt.apps.admin.warning.app_build.generic_secrets.one"
+      title: "App builder job: warning for generic secrets found in app text files"
+      value: "Possible secrets found in %{files}. Consider reviewing the contents of this file before submitting your app."
+  - translation:
+      key: "txt.apps.admin.warning.app_build.generic_secrets.other"
+      title: "App builder job: warning for generic secrets found in app text files"
+      value: "Possible secrets found in %{files}. Consider reviewing the contents of these files before submitting your app."
```

Comment:
Can you upload screenshots to Google Drive so that linguists will be able to see please?

https://zendesk.atlassian.net/wiki/spaces/globalization/pages/401277972/Upload+screenshots+to+Google+Drive